.github/workflows/ci.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     - run: make fmt
     - run: make gomodtidy
     - run: make fieldalignment
+    - run: make protolint
```

Comment:
Would it make sense to replace all the above with `make all`?

Reply:
That makes sense (note: it also runs `generate-modelpb`, which we didn't run before).

Reply:
:+1: that sounds desirable, it'll ensure that generated types are up to date.